example/server/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+
+from polaris.models import Transaction
+
+
+class PolarisUser(models.Model):
+    first_name = models.CharField(max_length=254)
+    last_name = models.CharField(max_length=254)
+    email = models.EmailField(unique=True)
+
+    objects = models.Manager()
+
+    @property
+    def name(self):
+        return " ".join([str(self.first_name), str(self.last_name)])
+
+
+class PolarisUserAccount(models.Model):
+    user = models.ForeignKey(PolarisUser, on_delete=models.CASCADE)
+    account = models.CharField(max_length=254, unique=True)
```

Comment:
should we set an exact length = 56 here since its always going to be a stellar public key just for some extra data integrity validation?